examples/llama/README.md

```diff
@@ -1,9 +1,41 @@
 # Llama
 
-## Get model
+## Clone this repo
 
 ```bash
-python -m onnxruntime_genai.scripts.export.py -m microsoft/phi-2 -e cpu -p int4 -o model/model.onnx
+git clone https://github.com/microsoft/onnxruntime-genai.git
+cd onnxruntime-genai
+```
+
+## Install the onnxruntime-genai library
+
+* (Temporary) Build the library according to [the build instructions](../README.md#build-from-source)
+
+* Install the python package
+
+  ```bash
+  cd build/wheel
+  pip install onnxruntime-genai-*.whl
+  ```
+
+
+## Get the model
+
+Install the model builder script dependencies
+
+```bash
+pip install numpy
+pip install transformers
+pip install torch
+pip install onnx
+pip install onnxruntime
+```
+
+Run the model builder script to load, optimize, quantize and export the model. More details can be found [here](../../src/python/py/models/README.md)
+
+```bash
+cd examples/phi2
+python -m onnxruntime_genai.models.builder.py -m meta-llama/Llama-2-7b-chat-hf -e cpu -p int4 -o ./model
```

Comment:
```suggestion
python -m onnxruntime_genai.models.builder -m meta-llama/Llama-2-7b-chat-hf -e cpu -p int4 -o ./llama2-7b-chat-int4-cpu
```

Reply:
i think you can also remove the model folders as the builder creates config files

Reply:
Removed